### PR TITLE
Revert "Revert "Use credentials tool to copy/decrypt credentials intead of referencing the secrets repo path""

### PR DESCRIPTION
**Pull request by jtreanor**
https://github.com/wordpress-mobile/WordPress-iOS/pull/12609 was reviewed and merged yesterday. Shortly after being merged @ScoutHarris encountered a problem where the credentials task wasn't being triggered, I reverted the change in wordpress-mobile/WordPress-iOS#12622.

This is a small addition to the original change to fix the issue @ScoutHarris was seeing. The problem was that the task checked for the presence of `~/.mobile-secrets/keys.json` which was only added relatively recently. It now checks for `~/.mobile-secrets/.git` which will be present for anyone who has the repo.

To test:

- Run `rake dependencies` and see that the `configure_apply` step is run.

### CODE REVIEW

File: Scripts/fastlane/Pluginfile [hunk 1]
```diff
@@ -6,5 +6,5 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 3.2.0'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.7.1'
+gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.7.2'
```
Comment (jtreanor) [inline Scripts/fastlane/Pluginfile]: 0.7.2 had a bunch of improvements to the apply behaviour: https://github.com/wordpress-mobile/release-toolkit/releases/tag/0.7.2

**Comment by ScoutHarris** [comment]
FWIW, I gave it a whirl. I did see `configure_apply` run and was able to log in.

**Comment by ScoutHarris** [comment]
Hey @jtreanor . I just noticed these files were created locally. Is this expected? 

new_files

**Comment by jtreanor** [comment]
@ScoutHarris yes, that is by design. They are also added to `.gitignore`

**Comment by ScoutHarris** [comment]
Hey @jtreanor .

> yes, that is by design. 

Um, maybe a dumb question - but I'm not sure how to "clear" these files. They keep getting recreated and there is a high probability I'll commit them accidentally. Halp! 😄

**Comment by jtreanor** [comment]
@ScoutHarris There is no need to clear the files. The reason you are seeing them is that you checked out this branch, ran `rake dependencies` and then switched to a branch where they were not ignored. This won't be a problem once this is merged (which I am doing now).

**Pull request merged**